ludwig/benchmarking/profiler.py

```diff
@@ -20,19 +22,38 @@
 from ludwig.globals import LUDWIG_VERSION
 from ludwig.utils.data_utils import save_json
 
-# disabling print because the following imports are verbose
-f = open(os.devnull, "w")
-sys.stdout = f
-from experiment_impact_tracker.cpu.common import get_my_cpu_info  # noqa E402
-from experiment_impact_tracker.gpu.nvidia import get_gpu_info  # noqa E402
-
-f.close()
-sys.stdout = sys.__stdout__
-
 STOP_MESSAGE = "stop"
 logger = logging.getLogger()
 
 
+def get_gpu_info():
+    """Gathers general hardware information about an nvidia GPU.
+
+    :param args:
```

Comment:
nit: Remove empty docs.

If this is copied from the other repo, please reference it as a comment i.e.
`Adapted from <link>, with additional adjustments for compatibility with Pandas 2.0.`